packages/webdriver/src/web-driver.ts

```diff
@@ -34,7 +40,7 @@ export function* connect(driver: WebDriver, options: Options): Operation<void> {
 
   let capabilities = new Atom(Capabilities);
 
-  if (options.headless) {
+  if (options.headless != null && options.headless) {
```

Comment:
is that an intentional `!=` instead of `!==`?

would `!!options.headless` work instead?

I would have the [eqeqeq](https://eslint.org/docs/rules/eqeqeq) rule set in eslint for strict equals

Reply:
I had done this originally so that specifying the boolean option wasn't _required_ so only do the check if it is not undefined, but I guess if it's undefined it will have the same effect as being false.